README.md

```diff
@@ -1,3 +1,278 @@
+<div align="center">
+  <a href="https://github.com/leapfrogtechnology/chill">
+    <img width="128px" src="opensource-logo.png"/> 
+  </a>
+  <br/>
+
 # opensource
 
 Opensource as a culture at Leapfrog Technology.
+
+# Featured List
+List of open source softwares in Leapfrog
```

Comment:
New line after the header, always.

```md
## Featured Open Source Projects
```

List of open source projects contributed and maintained by Leapfrog Technology.